docs/howto/jupyterhub.md

```diff
@@ -0,0 +1,75 @@
+## Log in to Wynton JupyterHub (JHUB)
+
+As of now JHUB can be accessed either by connecting to VPN or from campus network using [JupyterHub Login](https://jhub.wynton.ucsf.edu/hub/login).
+
+JHUB login is confined only to non-PHI Wynton user accounts. If you requested us to convert your account to a PHI account or setup your account as a PHI account, then you won't be able to login into JHUB.
```

Comment:
GRAMMAR: setup -> set up  (noun vs verb; I've done this mistake tons of times myself)